tests/test_slots.py

```diff
@@ -0,0 +1,26 @@
+from falcon import Request, Response
+import falcon.testing as testing
+
+
+class TestSlots(testing.TestBase):
```

Comment:
The `TestBase` class is deprecated; please use `falcon.testing.TestCase` instead.